src/cpp_common/utilities.cpp

```
@@ -0,0 +1,129 @@
+            {
+                base = "pgr_withPoints";
+                suffix = std::string(is_only_cost? "Cost" : "") + (is_matrix? "Matrix" : "");
+                break;}
+        default : base = "unknown";
+    }
+
+    return base + suffix;
+}
+
+char
+estimate_drivingSide(char driving_side, Which which) {
```

Cppcheck (2.19.0)

[style] The function 'estimate_drivingSide' is never used.

(unusedFunction)